exchanges/btcmarkets/btcmarkets.go

```diff
@@ -52,316 +60,482 @@ type BTCMarkets struct {
-		resp.Orders[i].Price /= common.SatoshisPerBTC
-		resp.Orders[i].OpenVolume /= common.SatoshisPerBTC
-		resp.Orders[i].Volume /= common.SatoshisPerBTC
+// GetTickers gets multiple tickers
+func (b *BTCMarkets) GetTickers(marketIDs string) ([]Ticker, error) {
+	var tickers []Ticker
+	arrayMarkets := strings.Split(marketIDs, ",")
+	params := url.Values{}
+	for x := range arrayMarkets {
+		params.Add("marketId", arrayMarkets[x])
+	}
+	path := btcMarketsUnauthPath + btcMarketsTickers + params.Encode()
+	return tickers, b.SendHTTPRequest(path, &tickers)
+}
 
-		for j := range resp.Orders[i].Trades {
-			resp.Orders[i].Trades[j].Fee /= common.SatoshisPerBTC
-			resp.Orders[i].Trades[j].Price /= common.SatoshisPerBTC
-			resp.Orders[i].Trades[j].Volume /= common.SatoshisPerBTC
+// GetMultipleOrderbooks gets orderbooks
+func (b *BTCMarkets) GetMultipleOrderbooks(marketIDs string) ([]Orderbook, error) {
+	var orderbooks []Orderbook
+	var temp []tempOrderbook
+	var tempOB Orderbook
+	arrayMarkets := strings.Split(marketIDs, ",")
+	params := url.Values{}
+	for x := range arrayMarkets {
+		params.Add("marketId", arrayMarkets[x])
+	}
+	path := btcMarketsUnauthPath + btcMarketsMultipleOrderbooks + params.Encode()
+	err := b.SendHTTPRequest(path, &temp)
+	if err != nil {
+		return orderbooks, err
+	}
+	for i := range temp {
+		var obData OBData
+		tempOB.MarketID = temp[i].MarketID
+		tempOB.SnapshotID = temp[i].SnapshotID
+		for a := range temp[i].Asks {
+			obData.Price, err = strconv.ParseFloat(temp[i].Asks[a][0], 64)
+			if err != nil {
+				return orderbooks, err
+			}
+			obData.Volume, err = strconv.ParseFloat(temp[i].Asks[a][1], 64)
+			if err != nil {
+				return orderbooks, err
+			}
+			tempOB.Asks = append(tempOB.Asks, obData)
 		}
+		for y := range temp[i].Bids {
+			obData.Price, err = strconv.ParseFloat(temp[i].Bids[y][0], 64)
+			if err != nil {
+				return orderbooks, err
+			}
+			obData.Volume, err = strconv.ParseFloat(temp[i].Bids[y][1], 64)
+			if err != nil {
+				return orderbooks, err
+			}
+			tempOB.Bids = append(tempOB.Bids, obData)
+		}
+		orderbooks = append(orderbooks, tempOB)
 	}
-
-	return resp.Orders, nil
+	return orderbooks, nil
 }
 
-// GetOpenOrders returns all open orders
-func (b *BTCMarkets) GetOpenOrders() ([]Order, error) {
-	type marketsResp struct {
-		Response
+// GetServerTime gets time from btcmarkets
+func (b *BTCMarkets) GetServerTime() (string, time.Time, error) {
+	var tempResp TimeResp
+	var t time.Time
+	path := btcMarketsAPIURL + btcMarketsAPIVersion + btcMarketsGetTime
+	err := b.SendHTTPRequest(path, &tempResp)
+	if err != nil {
+		return tempResp.Time, t, err
 	}
-	req := make(map[string]interface{})
-	var resp marketsResp
-	path := fmt.Sprintf("/v2/order/open")
-
-	err := b.SendAuthenticatedRequest(http.MethodGet, path, req, &resp)
+	t, err = time.Parse(time.RFC3339Nano, tempResp.Time)
 	if err != nil {
-		return nil, err
+		return tempResp.Time, t, err
 	}
+	return tempResp.Time, t, nil
+}
```

Comment:
Can make TimeResp a time.Time and json Unmarshal will take care of the parsing for you. Can also just return a time.Time var